CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## [*Ongoing*](https://github.com/pbrisbin/downgrade/compare/v6.3.0...master)
 
+- [NEW] Version bound handling by parsing logical =,>,<,>= and <= operators
+- [NEW] Handling basic version conflicts during downgrading
 - [NEW] Handling hard dependencies during downgrading
 
 ## [*v6.3.0*](https://github.com/pbrisbin/downgrade/compare/v6.1.0...v6.3.0)
```

Comment:
Same minor comment as #116: there might be a merge conflict here given that version headers are bolded, but in the previous PR they are not.